### ISSUE TITLE
Brand Refresh - Push Figma Brand Refresh Changes

### ISSUE THREAD

**Issue by vavalos5** [grommet/hpe-design-system#3239]
Follow on ticket to #3201 to push Figma changes

## Acceptance Criteria

Complete in order

- [x] Push typography and color changes
- [x] Pull in updates into each of our component files
- [x] Audit components that the changes are applied correctly post-push. Connect w/ dev if needed to align with Grommet components
- [x] Push all component changes
- [x] Review each file to ensure they've gone in properly
- [x] Replace imagery in migration guide with new changes
- [x] Review Figma changes one more time to ensure we didn't miss anything
- [x] Announce in #hpe-design-system channel and point people to Figma migration guide

**Comment by vavalos5** [comment]
3/31 - Kenny and I planning to push today and will be auditing after to ensure all of the typography and color styles go through.

**Comment by vavalos5** [comment]
4/4 - Kenny and I pushed color and typography styles. We've spent the last couple of days going through each file and ensuring things did properly go in. While it didn't for some files due to some being really old, we also took the time to ensure everything was in fact aligned to the theme and replaced deprecated or old components with correct ones.

I've updated the ticket above to reflect the other steps left to finalize the ticket. We feel strong about being able to complete by the end of the sprint.
